worlds/kh1/Options.py

```diff
@@ -155,6 +155,32 @@ class FinalRestDoor(Choice):
     option_postcards = 2
     option_superbosses = 3
 
+class RequiredPostcards(Range):
+    """
+    If your goal is set to "Postcards", defines how many postcards are needed to achieve victory.
+    """
+    display_name = "Required Postcards"
+    default = 10
+    range_start = 1
+    range_end = 10
+
+class RequiredPuppies(Choice):
+    """
+    If your goal is set to "Puppies", defines how many puppies are needed to achieve victory.
+    """
+    display_name = "Required Puppies"
+    default = 99
+    option_10 = 10
+    option_20 = 20
+    option_30 = 30
+    option_40 = 40
+    option_50 = 50
+    option_60 = 60
+    option_70 = 70
+    option_80 = 80
+    option_90 = 90
+    option_99 = 99
```

Comment:
Since the number of puppies in the pool cannot change, I think having options lower than 50 might be a bit of a mistake here.
If someone sets it to 10, they have ~900% more puppies in the pool than they need, which is really just useful as a joke option, and could cause some weirdly negative effects on a multiworld, like some people always finishing super super early. Which may be fine with you, just wanted to make sure that's understood.
Of course, requiring 99 also can have a pretty bad effect (requiring 100% of mcguffins is usually a bad call, but is fine to be there optionally).

I would recommend changing the default to 60 or 70 since that makes it ~50% extra mcguffins in the pool which is pretty standard. Maybe putting some warning about very low required puppy counts would be good too, but that's up to you of course.

Reply:
FWIW there are three different numbers of puppies possible because they can all be a single item (all 99 at once), each be an individual item, or come in triplets (33 items). Triplets is the default

Reply:
Is it not still 100% of mcguffins being the recommended default?

If you have it set to triplets and the goal amount is 10, you need 4 out of the 33 in the pool, which is a very very low amount.